### DISCUSSION TITLE
Any way to automatically generate Visual site map from our website or our existing sitemap.xml file using drawoi?

### DISCUSSION THREAD

**Discussion by merpdotcom** [jgraph/drawio-desktop#1336] [Q&A]
Greetings,

Is there any way to use drawio to automagically  _generate_ a visual sitemap either from our existing sitemap.xml file or by pointing it to our website and creating the visual sitemap?

We have many hundreds of pages and were hoping to autogenerate the visualization from our existing site.

We have a valid sitemap.xml file both on our side and downloaded. But I can't find information on any way to "import" this into drawio to dynamically create a visualization of the site structure.

It looks like more of the opensource visual site mapping tools have disappeared in the last 2-3 years, and visualsitemapper.com has been down since late May, and all the other sites charge.
Would prefer a self-hosted opensource solution, and hoping there is a way with drawio.

Any ideas?
Thanks kindly!

**Comment by davidjgraph** [comment]
No, this isn't supported in any way.
